bot/resources/evergreen/ryanzec_colours.json

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
You haven't moved this file yet.

Additionally, how did you get this file?

Reply:
Moved it!
I believe @CyberCitizen01 created it based on this: https://github.com/ryanzec/name-that-color/blob/master/lib/ntc.js#L116-L1681